dependencies.properties

```diff
@@ -30,6 +30,30 @@ jna-platform.key=CHK@G0F322jl0AxJqk-yfuRes5vPcpR~eb3pXjckqjr0qmU,mEDDjQHrPDt3Kt9
 jna-platform.size=2327597
 jna-platform.type=CLASSPATH
 jna-platform.order=3
+pebble.version=3.1.5
+pebble.filename=pebble-3.1.5.jar
```

Comment:
Is it worth moving to Pebble 3.1.6? https://github.com/PebbleTemplates/pebble/releases/tag/3.1.6

Reply:
"fix for older android" sounds like we have to …

Reply:
Actually no — the android build brings its own dependencies, to we can skip that for now.